{"name": "Неуспешный вход с неверным паролем", "status": "passed", "description": "Проверка ошибки при входе с неверным паролем", "steps": [{"name": "Авторизация пользователя: POST /api/auth/login", "status": "passed", "parameters": [{"name": "email", "value": "'elizabethbarker@example.com'"}, {"name": "password", "value": "'BadPassword123!'"}], "start": 1719860651603, "stop": 1719860652458}], "parameters": [{"name": "password", "value": "'BadPassword123!'"}], "start": 1719860651603, "stop": 1719860652459, "uuid": "25382043-ec76-4e5f-bb9a-9c7e78ce1688", "historyId": "99b0d8b19901924fa009630a16cf3035", "testCaseId": "0a64d4b6d1797ae683c0784dc1b12e04", "fullName": "tests.test_user.TestLoginUser#test_login_bad_password_failure", "labels": [{"name": "feature", "value": "Вход пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "NB-4179"}, {"name": "thread", "value": "11668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}